frontend/src/components/GovtMasthead/GovtMasthead.tsx

```diff
@@ -0,0 +1,121 @@
+import React from 'react'
+import { BiChevronDown, BiChevronUp } from 'react-icons/bi'
+import {
+  Box,
+  chakra,
+  Collapse,
+  Flex,
+  Icon,
+  Stack,
+  Text,
+  useDisclosure,
+} from '@chakra-ui/react'
+
+import { BxsBank } from '~assets/icons/BxsBank'
+import { BxsLockAlt } from '~assets/icons/BxsLockAlt'
+import Link from '~components/Link'
+
+import { GovtMastheadIcon } from './GovtMastheadIcon'
+
+// eslint-disable-next-line @typescript-eslint/no-empty-interface
+export interface GovtMastheadProps {}
```

Comment:
Should we remove the `GovtMastheadProps` interface if its empty altogether?

Reply:
done in 6934045